packaging/debian/postinst

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Does using `sh` instead make sense here?  I believe that on Ubuntu bash always exists (even if it's just a symlink to dash) but maybe it makes sense to say `sh` instead since it's more general?